### PR TITLE
Play 3.0 compatibility

### PR DESCRIPTION
**Pull request by sief**
- updated dependencies
- removed play-ahc-ws-standalone
- removed scala-collection-compat

### CODE REVIEW

**Comment by sief** [comment]
I didn't update the readme file because I didn't know how you were planning the next version numbering.

**Comment by julienrf** [comment]
Nice, thank you for your help? I guess the next release should be 7.0.0, no?

**Comment by sief** [comment]
Up to you, I just thought you might consider a separate versioning for Play 2 and Play 3.

File: README.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ Take a look at the [Scala](/sample-scala) and [Java](/sample-java) samples to se
...
 ```
 
-The current 6.0.0 version is compatible with Play 2.9 and Scala 2.13 and 3.3.
+The current 7.0.0 version is compatible with Play 3.0 and Scala 2.13 and 3.3.
 
 Previous versions are available here:
+ * [`6.0.0`](https://github.com/julienrf/play-jsmessages/tree/4.0.0) for play-2.9 ;
  * [`5.0.0`](https://github.com/julienrf/play-jsmessages/tree/4.0.0) for play-2.8 ;
  * [`4.0.0`](https://github.com/julienrf/play-jsmessages/tree/4.0.0) for play-2.7 ;
  * [`3.0.0`](https://github.com/julienrf/play-jsmessages/tree/3.0.0) for play-2.6 ;
```
Comment (julienrf) [inline README.md:23]:
```suggestion
 * [`6.0.0`](https://github.com/julienrf/play-jsmessages/tree/v6.0.0) for play-2.9 ;
 * [`5.0.0`](https://github.com/julienrf/play-jsmessages/tree/v5.0.0) for play-2.8 ;
 * [`4.0.0`](https://github.com/julienrf/play-jsmessages/tree/v4.0.0) for play-2.7 ;
 * [`3.0.0`](https://github.com/julienrf/play-jsmessages/tree/v3.0.0) for play-2.6 ;
```

**Comment by julienrf** [comment]
Thank you! :pray:

**Pull request merged**